### DISCUSSION TITLE
Save "Sort by" settings

### DISCUSSION THREAD

**Discussion by MajorTomDE** [community/community#39127] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hey

The default sorting of stars is "Recently starred".
For me it is more important to have "Recently active".

Currently "Sort by" is not saved. So I have to change "Sort by" every time I go to the Stars overview.

Please save "Sort by" settings permanent.

- For the general Stars list
- For the items within a list.

**Comment by RussellSwenson** [comment]
Travis

**Comment by jfullstackdev** [comment]
I'm using the regular bookmark of Chrome as of now, once you click your Sort by

or you can Save the link for that, since it's in the URL, like this 

https://github.com/jdevstatic?tab=repositories&q=&type=&language=&sort=stargazers

that's mine, get yours

![image](https://user-images.githubusercontent.com/47092464/202892613-f01275c4-c8c8-4b19-b83d-87bf3138ba16.png)
